cmd/node/config/gasSchedule.toml

```diff
@@ -1,3 +1,17 @@
+[BuiltInCost]
+    ChangeOwnerAddress    = 5000000
```

Comment:
this will lead to aprox 300 txs per block

Reply:
I did not evaluate the operation costs yet.